Sofa/GUI/Component/src/sofa/gui/component/performer/TopologicalChangeManager.cpp

```diff
@@ -290,6 +290,35 @@ Index TopologicalChangeManager::removeItemsFromPointModel(PointCollisionModel<so
     return res;
 }
 
+Index TopologicalChangeManager::removeItemsFromLineModel(LineCollisionModel<sofa::defaulttype::Vec3Types>* model, const type::vector<Index>& indices) const
+{
+    // EdgeSetTopologyContainer
+    sofa::core::topology::BaseMeshTopology* topo_curr;
+    topo_curr = model->getCollisionTopology();
+
+    if(dynamic_cast<EdgeSetTopologyContainer*>(topo_curr) == nullptr){
+        msg_warning("TopologicalChangeManager") << " Topology is not an EdgeSetTopologyContainer. Only EdgeSetTopologyContainer implemented.";
+        return 0;
+    }
+
+    // copy indices to have a mutable version of the vector
+    type::vector<Index> unique_indices = indices;
+    // sort followed by unique, to remove all duplicates
+    std::sort(unique_indices.begin(), unique_indices.end());
+    unique_indices.erase( std::unique(unique_indices.begin(), unique_indices.end()), unique_indices.end());
+
+    simulation::Node *node_curr = dynamic_cast<simulation::Node*>(topo_curr->getContext());
+    sofa::core::topology::TopologyModifier* topoMod;
+    topo_curr->getContext()->get(topoMod);
```

Comment:
auto topoMod = topo_curr->getContext()->get<sofa::core::topology::TopologyModifier*>()

Reply:
Also, if no modifier is providen in the scene, topoMod will be nullptr.
You should add a check on the pointer @ScheiklP 
Thanks @damienmarchal for the peer review.